cmd/bot/main.go

```diff
@@ -1,4 +1,52 @@
 package main
 
+import (
+	"fmt"
+	"github.com/Feyd-Rauth/Zlyden/pkg/zlyden"
+	"github.com/bwmarrin/discordgo"
+	"io/ioutil"
+	"log"
+	"os"
+	"os/signal"
+	"syscall"
+)
+
 func main() {
+	// loading Discord bot token
+	token, err := loadToken("token.dat")
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	discord, err := discordgo.New("Bot " + token)
+	if err != nil {
+		log.Fatal(err)
+	}
+
+	discord.AddHandler(zlyden.MessageCreate)
+
+	if err := discord.Open(); err != nil {
+		log.Fatal(err)
+	}
+	defer func(discord *discordgo.Session) {
+		err := discord.Close()
```

Comment:
This would probably result in error when closing session on "Say Goodbye". Is this intended?